virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -0,0 +1,266 @@
+
+[id="virt-about-cnv"]
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
+
+{VirtProductName} is represented by the image:virt-icon.png[{VirtProductName},40,40] icon.
+
+You can use {VirtProductName} with either the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+
+Learn more about xref:../../virt/about_virt/about-virt.adoc#about-virt[what you can do with {VirtProductName}].
+
+Learn more about xref:../../virt/about_virt/virt-architecture.adoc#virt-architecture[{VirtProductName} architecture and deployments].
+
+xref:../../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Prepare your cluster] for {VirtProductName}.
+
+include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+
+[id="virt-guest-os"]
+=== Supported guest operating systems
+To view the supported guest operating systems for {VirtProductName}, see link:https://access.redhat.com/articles/973163#ocpvirt[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization, OpenShift Virtualization and Red Hat Enterprise Linux with KVM].
+
+//Ensure platform passes Windows Server Virtualization Validation Program. Otherwise, comment out the section below.
+[id="virt-svvp-certification"]
+=== Microsoft Windows SVVP certification
+
+//CNV-31842 SVVP 4.15 Release Note: NEW
+//NOTE: This is a recurring release note. Modify the existing note text below if recommended by QE.
+{VirtProductName} is certified in Microsoft's Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
+
+The SVVP certification applies to:
+
+* Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS 9__.
+* Intel and AMD CPUs.
+
+[id="virt-quick-starts"]
+== Quick starts
+
+Quick start tours are available for several {VirtProductName} features. To view the tours, click the *Help* icon *?* in the menu bar on the header of the {VirtProductName} console and then select *Quick Starts*. You can filter the available tours by entering the `virtualization` keyword in the *Filter* field.
+
+
+[id="virt-4-15-new"]
+== New and changed features
+
+This release adds new features and enhancements related to the following components and concepts:
+
+[id="virt-4-15-installation-update"]
+=== Installation and update
```

Comment:
I was going to say something about this phrasing feeling awkward, but I see it's like this in the OCP RNs. Consistency is good!